### DISCUSSION TITLE
Full URL in Dynamic API Routes

### DISCUSSION THREAD

**Discussion by tuangeek** [vercel/next.js#20347] [Help]
Is it possible for an API route to detect a full URL in the query? For example I have `/pages/api/images/[...url].js` if I try to go to `localhost:3000/api/images/https://github.githubassets.com/images/modules/logos_page/GitHub-Logo.png` it returns a 404.

**Comment by paperclover** [comment]
i've been messing around with this and it seems the issue is the catch all doesn't catch double slashes (where it has https:**//** in the url), so i'm not too sure if thats a bug or what.
